parser.ts

```diff
@@ -223,43 +223,63 @@ export function traverseArguments(c: TreeCursor, s: string): Array<Expr<null>> {
   return args;
 }
 
+// Traverse the next target of an assignment and return it
+function traverseAssignment(c: TreeCursor, s: string): AssignTarget<null> {
+  let target = null;
+  let starred = false;
+  if (c.name === "*") {
+    // Check for "splat" starred operator
+    starred = true;
+    c.nextSibling();
+  }
+  try {
+    target = traverseExpr(c, s);
+  } catch (e) {
+    throw new Error(`Expected assignment expression, got ${s.substring(c.from, c.to)}`);
+  }
+  if (!isTagged(target, ASSIGNABLE_TAGS)) {
+    throw new Error(`Unknown target ${target.tag} while parsing assignment`);
+  }
+  let ignore = target.tag === "id" && target.name === "_"; // Underscores are ignored
```

Comment:
So I take it we'll still mark the `_` as ignored, but not actually do anything special about it? If that's the case, then maybe we should just remove the `ignore` field on `AssignTarget`. Either way is good with me, but if we leave it, maybe add a comment on the AST type(?)